gluster_exporter/metric_brick.go

```diff
@@ -167,6 +190,24 @@ func BrickUtilization() {
 					glusterBrickInodesTotal.With(lbls).Set(usage.InodesAll)
 					glusterBrickInodesFree.With(lbls).Set(usage.InodesFree)
 					glusterBrickInodesUsed.With(lbls).Set(usage.InodesUsed)
+					if usage.Used >= maxBrickUsed {
+						maxBrickUsed = usage.Used
+					}
+				}
+			}
+			if subvol.Type == "Arbiter" {
+				// In case of arbiter volume overall volume utilization
+				// will not have subvolume utilization and prometheus
+				// can use this scenario as alert condition as this would
+				// happen if all the data bricks are down
+				break
+			} else {
+				var subvolLabels = GetGlusterSubvolLabels(volume.Name, subvol.Name)
+				if subvol.Type == "Disperse" {
+					glusterSubvolCapacityUsed.With(subvolLabels).
```

Comment:
use one more variable so that function call can be added only once in the end.

    effectiveCapacity := maxBrickUsed
    var subvolLabels = GetGlusterSubvolLabels(volume.Name, subvol.Name)
    if subvol.Type == "Disperse" {
        // In disperse volume data bricks contribute to the sub volume size
        effectiveCapacity = maxBrickUsed * float64(subvol.DisperseDataCount)
    }

    // Export the metric only if available. it will be zero if the subvolume
    // contains only arbiter brick on current node or no local bricks on this node
    if effectiveCapacity > 0 {
        glusterSubvolCapacityUsed.With(subvolLabels).Set(effectiveCapacity)
    }